cppcache/integration/framework/Cluster.cpp

```diff
@@ -317,8 +317,9 @@ apache::geode::client::Cache Cluster::createCache(
     cacheFactory.set(property.first, property.second);
   }
 
-  auto cache = cacheFactory.set("log-level", "none")
+  auto cache = cacheFactory.set("log-level", "debug")
```

Comment:
This should not be part of your PR. Test should not log. Tests should assert behavior. Logging is a last resort for diagnosing uncertain failures.

Reply:
Meh - leftover from debugging test.  Will remove.